### PR TITLE
fix sample_classifier saplit_table

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qp_qiime2/qp_qiime2.py [hunk 1]
```diff
@@ -250,7 +250,7 @@ def call_qiime2(qclient, job_id, parameters, out_dir):
                 mkey = method_params[key]
                 # users can only select one value so if the view_type is set
                 # we will not convert
-                if mkey.view_type is not set:
+                if mkey.view_type not in (set, str):
```
Comment (charles-cowart) [inline qp_qiime2/qp_qiime2.py:253]: Just to confirm, it's okay if the key.view_type is either of type set or type str, and nothing else? They're two very different types.

**Review by charles-cowart — APPROVED** [review]

File: qp_qiime2/qp_qiime2.py (hunk 1 above)
Comment (antgonza) [reply qp_qiime2/qp_qiime2.py:253]: Yup, that's correct. Basically if set, that means that is really a string or set of strings but as Qiita only allows one value it becomes always a string. Then, if is set (one value AKA str) or if str, we don't need to cast the parameter and just pass it as is ...

Comment (charles-cowart) [reply qp_qiime2/qp_qiime2.py:253]: Ok.

**Comment by adswafford** [comment]
Verified working in Qiita this morning.

**Pull request merged**